cmd/integration_test.go

```diff
@@ -1,4 +1,4 @@
-// +build integration
```

Comment:
It's very 😢 that these don't run in Travis and by default locally since they represent almost the only test coverage of the core logic here... it looks like this change breaks the tests on Travis though.

Reply:
I totally agree :/